java/core/src/main/java/co/worklytics/psoxy/gateway/impl/oauth/CanonicalOAuthAccessTokenResponseDto.java

```diff
@@ -30,7 +30,8 @@
     String accessToken;
     String refreshToken;
     String tokenType;
-    Integer expires;
+    //OAuth 2.0 https://datatracker.ietf.org/doc/html/rfc6749#section-4.4.3
```

Comment:
Maybe add a `@JsonAlias("expires")` ? What was the use case of "expires" ?

Reply:
i don't know. it's not in the spec. did you see a data source that returns it? I think maybe some return a timestamp for `expires`, instead of `expires_in`, which is TTL in seconds.